src/Integration/Binding/IRuleSetGenerationFileSystem.cs

```diff
@@ -10,16 +10,10 @@
 namespace SonarLint.VisualStudio.Integration.Binding
 {
     // Interface for testing purposes only
-    internal interface IRuleSetGenerationFileSystem
+    internal interface IRuleSetFileSystem
```

Comment:
Separate out to IFileSystem for the basic stuff and IRuleSetFileSystem for the ruleset specific operations.